ygot/struct_validation_map.go

```diff
@@ -820,6 +820,10 @@ func copySliceField(dstField, srcField reflect.Value, opts ...MergeOpt) error {
 	}
 
 	if _, ok := srcField.Interface().([]Annotation); !ok {
+		if diff := cmp.Diff(srcField.Interface(), dstField.Interface()); diff == "" {
```

Comment:
Does `cmp.Diff` do something different to `cmp.Equal` here? I can't remember why we used `cmp.Diff` originally.

Reply:
https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff
```
It returns an empty string if and only if Equal returns true for the same input values and options.
```

Reply:
Given that you don't use the `diff` here, I suggest just changing this to be `if !cmp.Equal()`.